docs/modules/circulation/pages/booking.adoc

```diff
@@ -1,180 +1,231 @@
-actions button. Select "Create Reservation".
-. Enter the patron's barcode.
-. Adjust the values in this screen as necessary.
-. Select the "Confirm reservation" button.
-. The screen will refresh, and the new reservation will appear in the schedule.
-
+The "Create a booking reservation" screen uses your library's timezone.
+If you create a reservation at a library in a different timezone, Evergreen
+will alert you and provide the time in both your timezone and the other
+library's timezone.
+
+Reservations can only be created by staff members and can be initiated from
+a patron record, a booking resource, or the catalog if the booking item's
+barcode is not known.
+
+=== From a Patron Record ===
+
+1. Retrieve the patron's record.
+2. Click **Other → Booking: Create Reservations**.
+   This opens the Create Reservations screen.
+
+.Other → Booking: Create Reservations
+image::booking/booking-create-from-patron-1.png[Drop-down menu showing the path to create reservations from a patron record.,width=250]
+
+[start=3]
+3. Choose reservation type and date:
+
+.Reservation Type and Date Fields
+image::booking/booking-create-reservation-type-date.png[Reservation type and date fields., width=650]
+
+    - **Single-day reservation**: For reservations lasting less than a day
+      (e.g., a study room).
+    - **Multiple-day reservation**: For reservations spanning several days
+      (e.g., a video camera).
+
+[start=4]
+4. Specify the resource:
+
+.Reservation Details Fields
+image::booking/booking-create-reservation-details.png[Reservation details fields.,width=650]
+
+    - **Choose resource by barcode**: If you know the barcode.
+    - **Choose resource by type**: To select by type if the barcode is
+    unknown.
+    Resource types are managed in the Booking Administration module. See <<_additional_resources,Additional Resources>> above.
+
+[start=5]
+5. Adjust the schedule grid and settings as needed:
+    - Use the **Schedule settings** tab for day/time adjustments.
+    - Specify any additional resource attributes in the **Attributes** tab
+    (e.g., PC vs.
+    Mac laptop).
+
+.Schedule Grid
+image::booking/booking-schedule-grid.png[Schedule grid showing available times., width=350]
+
+[start=6]
+6. Create the reservation using one of the following:
+    - Double-click the appropriate row in the schedule grid.
+    - Use keyboard navigation and the **Actions** menu (Shift+F10).
+    - Right-click the row and choose **Create Reservation**.
+    - Click the **Actions** button and select **Create Reservation**.
+
+[start=7]
+7. Adjust reservation details as necessary:
+    - Patron barcode
+    - Start/End times
+    - Notify patron by email (optional)
+    - Notes (optional)
+    
+.Confirm Reservation Modal
+image::booking/booking-confirm-reservation.png[Confirm reservation modal showing reservation details.,width=650]
+
+[start=8]
+8. Click **Confirm Reservation**.
+   The screen refreshes, displaying the new reservation in the schedule.
```

Comment:
Rather than hardcoding the list index, [could you please rely on Asciidoc's automatic numbering](https://docs.asciidoctor.org/asciidoc/latest/lists/ordered/)?  This will make it easier to add/remove items in the future without needing to renumber everything.  To insert images in the middle of the list, you can [precede it with a plus sign on a line by itself to mark it as a continuation of the current list item](https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/#list-continuation).

```suggestion
. Retrieve the patron's record.
. Click **Other → Booking: Create Reservations**.
   This opens the Create Reservations screen.
+
.Other → Booking: Create Reservations
image::booking/booking-create-from-patron-1.png[Drop-down menu showing the path to create reservations from a patron record.,width=250]
. Choose reservation type and date:
+
.Reservation Type and Date Fields
image::booking/booking-create-reservation-type-date.png[Reservation type and date fields., width=650]
    - **Single-day reservation**: For reservations lasting less than a day
      (e.g., a study room).
    - **Multiple-day reservation**: For reservations spanning several days
      (e.g., a video camera).
. Specify the resource:
+
.Reservation Details Fields
image::booking/booking-create-reservation-details.png[Reservation details fields.,width=650]

    - **Choose resource by barcode**: If you know the barcode.
    - **Choose resource by type**: To select by type if the barcode is
    unknown.
    Resource types are managed in the Booking Administration module. See <<_additional_resources,Additional Resources>> above.
. Adjust the schedule grid and settings as needed:
    - Use the **Schedule settings** tab for day/time adjustments.
    - Specify any additional resource attributes in the **Attributes** tab
    (e.g., PC vs.
    Mac laptop).
+
.Schedule Grid
image::booking/booking-schedule-grid.png[Schedule grid showing available times., width=350]
. Create the reservation using one of the following:
    - Double-click the appropriate row in the schedule grid.
    - Use keyboard navigation and the **Actions** menu (Shift+F10).
    - Right-click the row and choose **Create Reservation**.
    - Click the **Actions** button and select **Create Reservation**.
. Adjust reservation details as necessary:
    - Patron barcode
    - Start/End times
    - Notify patron by email (optional)
    - Notes (optional)
+
.Confirm Reservation Modal
image::booking/booking-confirm-reservation.png[Confirm reservation modal showing reservation details.,width=650]
. Click **Confirm Reservation**.
   The screen refreshes, displaying the new reservation in the schedule.
```